include/SANDTrackerCell.h

```diff
@@ -0,0 +1,109 @@
+#pragma once
+
+#include "SANDWireInfo.h"
+
+class SANDTrackerPlane;
+
+class SANDTrackerCellID : public SingleElStruct<unsigned long>
+{
+ public:
+  SANDTrackerCellID(unsigned long id) : SingleElStruct<unsigned long>(id){};
+  SANDTrackerCellID() : SingleElStruct<unsigned long>(){};
+};
+
+class SANDTrackerCell
+{
+  SANDTrackerCellID _id;
+  SANDWireInfo _wire;
+  double _width;
+  double _height;
+  double _driftVelocity;  // drift velocity in um/ns
+
+  SANDTrackerPlane* _plane;
+  std::vector<SANDTrackerCell*> _adjacent_cells;
+
+ public:
+  SANDTrackerCell() {};
+  SANDTrackerCell(const SANDTrackerCellID cID,
+                  const SANDWireInfo &l, 
+                  const double w,
```

Comment:
`const double w`   -> `double w`

cheap to copy

https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#f16-for-in-parameters-pass-cheaply-copied-types-by-value-and-others-by-reference-to-const

this comment applies to other functions defined here.